.github/pull_request_template.md

```diff
@@ -1,17 +1,8 @@
-### What does this PR do?
+### What this PR does / why we need it
 
-(Please set a descriptive PR title. Use this space for additional explanations.)
+<!-- Please mention any GitHub issue, set a descriptive PR title, and use this space for additional explanations. -->
 
-### What does it look like?
+### Checklist
 
-(If this is more than a textual change, a screenshot can help in some cases to speed up the review process.)
-
-### Any background context you can provide?
-
-(Please link public issues or summarize if not public.)
-
-### What is needed from the reviewers?
-
-### Have you maintained the front matter?
-
-(Please bump the last_review_date in case this qualifies as a review for an entire page. Provide user_questions which should be answered in the page. Provide a meaningful description.)
+- [ ] (Content changes only) Ran `make dev` to test content changes locally.
+- [ ] (Content changes only) Bumped `last_review_date` in the front matter header because I reviewed the entire page.
```

Comment:
Again, it's fuzzy whether we want this on every docs PR now. I recommend against it and so would not make it part of a checklist.

Reply:
Yes, both items are optional. Not great, I know. At least `make dev` is _recommended_, and the review date bump is _conditional_. If we don't mention it here, nobody will take care of that or know about it. Which alternatives do we have here? One idea is to make it a bullet point list "Things to check/remember" instead of items to be ticked off.

Reply:
> One idea is to make it a bullet point list "Things to check/remember" instead of items to be ticked off.

Yes, I like that better.

Reply:
Changed to bullet points